Source/DataTypes/SvgUnitConverter.cs

```diff
@@ -1,120 +1,193 @@
 using System;
 using System.ComponentModel;
 using System.Globalization;
+using Svg.Helpers;
 
 namespace Svg
 {
     public sealed class SvgUnitConverter : TypeConverter
     {
-        public override object ConvertFrom(ITypeDescriptorContext context, System.Globalization.CultureInfo culture, object value)
+        public static SvgUnit Parse(ReadOnlySpan<char> unit)
         {
-            if (value == null)
-            {
-                return new SvgUnit(SvgUnitType.User, 0.0f);
-            }
-
-            if (!(value is string))
-            {
-                throw new ArgumentOutOfRangeException("value must be a string.");
-            }
-
             // http://www.w3.org/TR/CSS21/syndata.html#values
             // http://www.w3.org/TR/SVG11/coords.html#Units
 
-            string unit = (string)value;
             int identifierIndex = -1;
 
-            if (unit == "none")
+            if (unit.SequenceEqual("none".AsSpan()))

```

Comment:
wouldn't it be better to have a readonly variable with the value of

`private static readonly ReadOnlySpan<char> none= "none".AsSpan();`

and than compare 

`unit.SquenceEqual(none)`

Reply:
Not possible, you can only define ReadOnlySpan as field in `ref struct` not in class.

Compiler will give you error:
```
  SvgUnitConverter.cs(10, 33): [CS8345] Pole lub automatycznie implementowana właściwość nie może być typu „ReadOnlySpan<char>”, chyba że jest to składowa struktury ref.
```